packages/core/src/filters/FilterSystem.ts

```diff
@@ -319,27 +319,35 @@ export class FilterSystem extends System
 
     /**
      * Binds a renderTexture with corresponding `filterFrame`, clears it if mode corresponds.
+     *
      * @param {PIXI.RenderTexture} filterTexture - renderTexture to bind, should belong to filter pool or filter stack
      * @param {PIXI.CLEAR_MODES} [clearMode] - clearMode, by default its CLEAR/YES. See {@link PIXI.CLEAR_MODES}
      */
     bindAndClear(filterTexture: RenderTexture, clearMode = CLEAR_MODES.CLEAR): void
     {
+        const {
+            renderTexture: renderTextureSystem,
+            state: stateSystem,
+        } = this.renderer;
+
         if (filterTexture && filterTexture.filterFrame)
         {
             const destinationFrame = this.tempRect;
 
             destinationFrame.width = filterTexture.filterFrame.width;
             destinationFrame.height = filterTexture.filterFrame.height;
 
-            this.renderer.renderTexture.bind(filterTexture, filterTexture.filterFrame, destinationFrame);
+            renderTextureSystem.bind(filterTexture, filterTexture.filterFrame, destinationFrame);
         }
         else
         {
-            this.renderer.renderTexture.bind(filterTexture);
+            renderTextureSystem.bind(filterTexture);
         }
 
```

Comment:
heya! would be great to comment this line, as its hard to tell whats going on 🥇

Reply:
This line:
```js
const autoClear = (stateSystem.stateId & 1) || this.forceClear;
```

Reply:
I'll add a comment when the workday is over (be patient, don't merge this until I do pls even tho it has 2 approvals)

Reply:
Roger that!